client/src/_dummies/category.json

```
@@ -0,0 +1,10 @@
+{
+  "categories": [
+    { "id": 1, "name": "생활", "value": "life", "color": "" },
+    { "id": 2, "name": "식비", "value": "food", "color": "" },
+    { "id": 3, "name": "교통", "value": "transportation", "color": "" },
+    { "id": 4, "name": "쇼핑/뷰티", "value": "beauty", "color": "" },
+    { "id": 4, "name": "의료", "value": "medical", "color": "" },
+    { "id": 5, "name": "건강", "vlaue": "healthy", "color": "" }
+  ]
```

```suggestion
  "categories": [
    { "id": 1, "name": "생활", "value": "life", "color": "#4A6CC3" },
    { "id": 2, "name": "식비", "value": "food", "color": "#4CA1DE" },
    { "id": 3, "name": "교통", "value": "transportation", "color": "#94D3CC" },
    { "id": 4, "name": "쇼핑/뷰티", "value": "beauty", "color": "#4CB8B8" },
    { "id": 5, "name": "의료/건강", "value": "healthy", "color": "#6ED5EB" },
    { "id": 6, "name": "문화/여가", "value": "culture", "color": "#D092E2" },
    { "id": 7, "name": "미분류", "value": "else", "color": "#817DCE" }
  ]
```

이 부분은 `constants`로 관리해도 되지 않을까요? 아니면 추후 데이터베이스에 저장하나요 ?

---

네, DB로 가게 될 거 같아요!
 카테고리 테이블도 있고 히스토리 테이블이 참조하고 있어요!

---

그럼 color와 value 값도 DB에 저장하나용?

---

관리자 페이지로 관리한다고 가정하면 DB에 있는게 맞을 거 같아요!
클라이언트에 붙어 있으면 태그 추가/수정/삭제 할 때 마다 개발자가 빌드 해줘야 하니까요! 😢 
물론 그냥 가정한 상황이니까...!ㅋㅋㅋㅋ 사실 안 해도 됩니다요!

---

오호 그르네요 !! 완전 이해했습니다 !! 컬러값을 어디에서 관리해야하지? 하는 의문이 있었는데 생각해보니 태그 추가/수정/삭제가 일어날 경우 프로세스가 복잡해지겠네요 !! 좋습니다 !